### DISCUSSION TITLE
Supabase installation fails

### DISCUSSION THREAD

**Discussion by navingate** [elizaOS/eliza#421] [Q&A]
Hello all,
I'm trying enable Supabase for the project. I'm following the steps mentioned here - https://github.com/Ungate-Ai/chitti2.0/blob/main/docs/docs/packages/adapters.md

Issues I'm facing - 

1. pnpm add @ai16z/adapter-supabase @supabase/supabase-js - does not work. Instead,  - pnpm add “@ai16z/adapter-supabase” @supabase/supabase-js - works
2. Even with this, when i add "import { SupabaseDatabaseAdapter } from "@ai16z/adapter-supabase";" to /packages/agent/src/index.ts, the import does not resolve @ai16z/adapter-supabase

Has anyone else faced this before? Appreciate any support

**Comment by 0xVitae** [comment]
Lot's of people running into issue of setting up supabase in eliza
